### DISCUSSION TITLE
Empty prediced values at 68th Error

### DISCUSSION THREAD

**Discussion by noritoshitamura** [PaddlePaddle/PaddleSpatial#140] [BaiduKDDCup2022] [answered]
I got a error. 

![sc](https://user-images.githubusercontent.com/101804962/171778181-c02a70c9-b59f-49c7-b23a-f792efd92580.png)

Are there  something special  at 68th prediction ?

**Comment by ramber1836** [comment]
Please provide your team name.

**Reply by noritoshitamura** [reply]
Team name is noritoshiTeam.
Thank's.

**Reply by ramber1836** [reply]
![image](https://user-images.githubusercontent.com/19561583/171781632-c6afc2f8-19ca-43e9-9f44-9128db494c13.png)

Nothing special for prediction the 68th prediction.

**Comment by ChaoqunSu** [comment]
> I got a error.
> 
> ![sc](https://user-images.githubusercontent.com/101804962/171778181-c02a70c9-b59f-49c7-b23a-f792efd92580.png)
> 
> Are there something special at 68th prediction ?
 
I ran into the same problem as you, which is the 68th prediction.  Have you solved the problem yet??

**Reply by noritoshitamura** [reply]
I'm not yet.
I wonder why ,it occurred at 68th prediction ? It looks like data problem...
Any way,  I believe the data and will  add a logic for cleaning up objects in each prediction to prevent resource exhaust.

**Comment by ramber1836** [comment, marked as answer]
@noritoshitamura @ChaoqunSu 
The online test has been updated for the empty prediction check. Please resubmit your code and try it again. Thanks.
